### ISSUE TITLE
unable to access tixiSetPrintMsgFunc

### ISSUE THREAD

**Issue by nvanoene** [DLR-SC/tixi#77]
I'm trying to use the tixiSetPrintMsgFunc to stop tixi from printing warnings. However, only the GetPrintMsgFunc is available so I am unable to change the output as said in the documentation.

Sincerely,
Nick

**Comment by rainman110** [comment]
Hi Nick, 

what do you mean with unavailable? We are using this function from our TiGL library 😏 

If you are using another language than C, yes it's not available. The reason is, that we don't know, how to make callbacks into other languages right know.

If you just want to change the verbosity of the warnings, we could also add a function that allows you to adjust it.

**Comment by nvanoene** [comment]
Dear Mr. Siggel,

I'm doing my internship at DLR and I'm using CPACS files as input and use
the tixi library to process them. I'm working in python.
Currently when I run my program, tixi keeps printing warnings in the
console and I indeed want to change the verbosity (of the logging then?) of
tixi such that it stops printing them.
In the online tixi documentation I found the SetPrintMsgFunc which should
allow me to do so, so hence my question. If it would be possible to have a
function in tixi to change the verbosity that would be really great.
I know that such a function exists for TiGL already.

Sincerely,
Nick

2016-10-20 15:18 GMT+02:00 Martin Siggel <notifications@github.com>:
